Env variable for restricted code execution (#2598)
databricks/cli: NEXT_CHANGELOG.md

### Bundles
* Fixed cleaning up artifact path .internal folder ([#2572](https://github.com/databricks/cli/pull/2572))
* Added support for quality monitors in deployment bind/unbind commands ([#2583](https://github.com/databricks/cli/pull/2583))

> * New DATABRICKS_BUNDLE_RESTRICTED_CODE_EXECUTION environment variable which disables running scripts and python code when running bundle commands ([#2598](https://github.com/databricks/cli/pull/2598))

Reviewer: ```suggestion
* New DATABRICKS_BUNDLE_RESTRICTED_CODE_EXECUTION environment variable that rejects running scripts and Python code when it is set ([#2598](https://github.com/databricks/cli/pull/2598))
```
